engine/src/main/scala/cromwell/engine/db/DataAccess.scala

```diff
@@ -148,7 +148,7 @@ trait DataAccess extends AutoCloseable {
   def queryWorkflowOutputs(id: WorkflowId)
                           (implicit ec: ExecutionContext): Future[Seq[MetadataEvent]] = {
     val uuid = id.id.toString
-    queryMetadataEventsWithWildcardKeys(uuid, s"${WorkflowMetadataKeys.Outputs}:".wrapNel, requireEmptyJobKey = true).
+    queryMetadataEventsWithWildcardKeys(uuid, s"${WorkflowMetadataKeys.Outputs}:%".wrapNel, requireEmptyJobKey = true).
```

Comment:
I don't think it was working before.  I just ran a workflow on a cromwell server off develop and the /outputs endpoint just returns the workflow id and no outputs (even though /metadata has it)